static/src/javascripts/projects/common/modules/video/events.js

```diff
@@ -109,36 +110,50 @@ define([
         bindCustomMediaEvents(eventsMap, player, mediaId, mediaType, true);
     }
 
+    function getGoogleAnalyticsEventAction(mediaEvent) {
+        var action = mediaEvent.mediaType + ' ';
+        if (mediaEvent.isPreroll) {
+            action += 'preroll';
+        } else {
+            action += 'content';
+        }
+        return action;
+    }
+
+    function buildGoogleAnalyticsEvent(mediaEvent, metrics, canonicalUrl) {
+        var category = 'Media';
+        var playerName = 'guardian-videojs';
+        var action = getGoogleAnalyticsEventAction(mediaEvent);
+        var fieldsObject = {
+            eventCategory: category,
+            eventAction: action,
+            eventLabel: canonicalUrl,
```

Comment:
@akash1810 It seems to be missing for quite a lot of videos as well. Not sure why.

![screen shot 2016-08-08 at 12 26 29](https://cloud.githubusercontent.com/assets/106760/17478404/698d24aa-5d63-11e6-963e-4a3c219243f2.png)